codegen/templates/endpoint.tmpl

```diff
@@ -101,7 +102,22 @@ func (h *{{$handlerName}}) HandleRequest(
 	req *zanzibar.ServerHTTPRequest,
 	res *zanzibar.ServerHTTPResponse,
 ) {
-	{{- if $reqHeaderRequiredKeys -}}
+	defer func() {
+		if r := recover(); r != nil {
+			stacktrace := string(debug.Stack())
+			e := errors.Errorf("enpoint panic: %v, stacktrace: %v", r, stacktrace)
+			h.Dependencies.Default.ContextLogger.Error(
```

Comment:
Can you also increment a specific metric for panics so we can add alerting on specifically panics?

Reply:
fixed